sonic-net/sonic-buildimage
# [DPB] Dell Z9332f port breakout changes

#### Why I did it
Added DPB support in Dell Z9332f
#### How I did it
Edited platform and hwsku files
#### How to verify it
Execute breakout commands and verify the status in "show interface status".
[DPB UT.txt](https://github.com/sonic-net/sonic-buildimage/files/10058575/DPB.UT.txt)

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
